### PR TITLE
@broskoski => little cleanup and use lot standing instead of iterating through bidder positions

### PR DESCRIPTION
**Pull request by sweir27**
This PR follows https://github.com/artsy/force/pull/128 :)

Since there's a gravity endpoint to return your bidding status based on a given lot, we no longer have to iterate through bidder positions to figure out whether or not you're the highest bidder.

### CODE REVIEW

File: apps/artwork/components/bid/query/me.coffee [hunk 1]
```diff
@@ -1,8 +0,0 @@
-module.exports = """
```
Comment (sweir27) [inline apps/artwork/components/bid/query/me.coffee:1]: Seems like we don't need this anymore given that we never reference `me.bidder_positions`

File: models/current_user.coffee [hunk 2]
```diff
@@ -43,24 +43,6 @@ module.exports = class CurrentUser extends Backbone.Model
         options.success(false) if xhr.responseText?.match 'A user is required'
         options.error? arguments...
 
-  # Checks whether a user is the highest bidder for a given SaleArtwork.
-  #
-  # @param {String} saleArtwork
-  # @param {Object} options Provide `success` and `error` callbacks, successes with true or false
-
-  highestBidderFor: (saleArtwork, options = {}) ->
```
Comment (sweir27) [inline models/current_user.coffee:51]: This method didn't seem to be used anywhere!

File: apps/artwork/components/bid/query/me.coffee (hunk 1 above)
Comment (broskoski) [reply apps/artwork/components/bid/query/me.coffee:1]: 👍

File: apps/artwork/components/bid/templates/bid.jade [hunk 3]
```diff
@@ -12,8 +12,8 @@
       span Reserve not met
     else
       span No reserve
-  if bidder && bidder.bidder_positions.length > 0 && typeof window !== 'undefined'
-    if _.find(bidder.bidder_positions, { is_winning: true })
+  if bidder && bidder.lot_standing && typeof window !== 'undefined'
+    if bidder.lot_standing.is_highest_bidder
```
Comment (broskoski) [inline apps/artwork/components/bid/templates/bid.jade:16]: Ah, much nicer

File: models/current_user.coffee (hunk 2 above)
Comment (broskoski) [reply models/current_user.coffee:51]: 👍

**Pull request merged**